src/main/scala/Keyboard.scala

```diff
@@ -0,0 +1,37 @@
+object Keyboard extends App {
+  import scala.io.Source
+  import scala.io.StdIn._
+  println("Enter number of character strings to read")
+  val wordAmount = readInt
+  var stringsArray = new Array[String](wordAmount)
+  println("Type that many strings")
+  for (i <- 0 until wordAmount)stringsArray(i) = readLine
+
+  val dictionary = Source.fromFile("C:\\Users\\Admin\\Desktop\\Scala\\Material\\enable1.txt").getLines.toArray
+
+  var matchCount = 0
+  var validCharCount = 0
+  var highCount=0
+  var longestWord = ""
+  var validWord = 0
+
+  for (whichWord <- 0 until wordAmount) {
```

Comment:
I honestly don't get the logic here, it's way too complex... does it actually work?